### PR TITLE
feat: Display message deleted

### PR DESCRIPTION
**Pull request by fredrikleknes**

### CODE REVIEW

File: datasheetapi/Services/CommentService.cs [hunk 1]
```diff
@@ -174,6 +174,7 @@ public async Task DeleteComment(Guid id, Guid azureUniqueId)
 
         existingComment.Text = updatedComment.Text;
         existingComment.IsEdited = true;
+        existingComment.SoftDeleted = updatedComment.SoftDeleted;
```
Comment (taustad) [inline datasheetapi/Services/CommentService.cs:177]: This essentially means you can undelete a comment if you send the correct PUT request. Is this something we want?

Comment (fredrikleknes) [reply datasheetapi/Services/CommentService.cs:177]: I was thinking if we at some point wanted an 'undo' option? But probably not, should i remove it?

Comment (taustad) [reply datasheetapi/Services/CommentService.cs:177]: I think we should return "comment deleted" if you try to update a deleted comment. Either that or we need to implement the undo function. 

@DanielBohme @manikandan5027 , opinions?

Comment (DanielBohme) [reply datasheetapi/Services/CommentService.cs:177]: An "undo" option seems weird to me after one has deleted their comment. I would rather have a pop up when deleting for "are you sure you want to delete this?" or something like that.

Comment (fredrikleknes) [reply datasheetapi/Services/CommentService.cs:178]: Added an exception for this. Also, the option to delete does not show in frontend when comment is already deleted.
Is this a fine solution? @taustad @DanielBohme

Comment (taustad) [reply datasheetapi/Services/CommentService.cs:178]: Move the check to right after retrieving the existing comment

**Review by taustad — APPROVED** [review]

**Review by taustad — APPROVED** [review]

**Comment by fredrikleknes** [comment]
AB#123431

**Comment by fredrikleknes** [comment]
AB#123793

**Pull request merged**